### PR TITLE
#156 - added IT and dummy slice for deprecate command

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #156 
Adde IT for `npm deprecate` and dummy (for now) slice.

### CODE REVIEW

File: src/main/java/com/artipie/npm/http/NpmSlice.java [hunk 1]
```diff
@@ -116,12 +122,26 @@ public NpmSlice(
...
+                    new UploadSlice(npm, storage),
+                    auth,
+                    new Permission.ByName(perms, Action.Standard.WRITE)
+                )
+            ),
+            new RtRulePath(
+                new RtRule.All(
+                    new ByMethodsRule(RqMethod.PUT),
+                    new RtRule.ByHeader(NpmSlice.REFERER, Pattern.compile("deprecate.*"))
+                ),
```
Comment (genryxy) [inline src/main/java/com/artipie/npm/http/NpmSlice.java:139]: @olenagerasimova probably to move pattern for deprecate to `DeprecateSlice`

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one suggestion

**Comment by olenagerasimova** [comment]
@genryxy thanks, corrected

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, approve

**Comment by genryxy** [comment]
@g4s8 could you please check/merge

File: src/main/java/com/artipie/npm/http/DeprecateSlice.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import java.nio.ByteBuffer;
+import java.util.Map;
+import java.util.regex.Pattern;
+import org.reactivestreams.Publisher;
+
+/**
+ * Slice to handle `npm deprecate` command requests.
+ * @since 0.8
+ */
+public final class DeprecateSlice implements Slice {
```
Comment (g4s8) [inline src/main/java/com/artipie/npm/http/DeprecateSlice.java:38]: @olenagerasimova do we really need a class just to keep some static field?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova please see my comment

File: src/main/java/com/artipie/npm/http/DeprecateSlice.java (hunk 2 above)
Comment (olenagerasimova) [reply src/main/java/com/artipie/npm/http/DeprecateSlice.java:38]: @g4s8 this class will be properly implemented in the next PR

**Review by g4s8 — APPROVED** [review]
@olenagerasimova got it, thanks

**Comment by 0crat** [comment]
Job `gh:artipie/npm-adapter#166` is not assigned, can't get performer

**Pull request merged**